packages/ui/src/components/Menu/Menu.module.css

```diff
@@ -1,72 +1,106 @@
 .aside {
-    position: fixed;
-    z-index: 99;
-    top: var(--header-height);
-    left: 0;
-    bottom: 0;
-    width: var(--menu-width);
-    background: linear-gradient(
-            to bottom,
-            hsl(217, 22%, 20%),
-            hsl(217, 22%, 16%) 80%,
-            hsl(217, 22%, 12%)
-    );
-    padding-top: 1rem;
-    color: #d5d9dc;
-    display: flex;
-    flex-direction: column;
-    box-shadow: 4px 0 8px 3px rgba(38, 46, 60, 0.1);
+  position: fixed;
+  z-index: 99;
+  top: var(--header-height);
+  left: 0;
+  bottom: 0;
+  width: var(--menu-width);
+  background: linear-gradient(
+    to bottom,
+    hsl(217, 22%, 20%),
+    hsl(217, 22%, 16%) 80%,
+    hsl(217, 22%, 12%)
+  );
+  padding-top: 1rem;
+  color: #d5d9dc;
+  display: flex;
+  flex-direction: column;
+  box-shadow: 4px 0 8px 3px rgba(38, 46, 60, 0.1);
 }
 
 .aside > div {
-    color: #828e97;
-    font-size: 0.833em;
```

Comment:
please enable prettier at you coding env, so it will align styling

Reply:
Hm, my prettier auto-formatted the file to change the 4 spaces to 2. Not sure why.

Reply:
I think there's no tabWidth set specifically for CSS files in the `prettier.config.js`, do you have local config for that?

Reply:
Your are correct, this file was missed out...